tests/utils/test_utilities.py

```diff
@@ -0,0 +1,29 @@
+from xgi.utils import XGICounter, get_dual, load_xgi_data
+
+
+def test_load_xgi_data():
```

Comment:
I'm hesitant to make an HTTP request part of the test suite. Perhaps we could mark it as [slow](https://docs.pytest.org/en/6.2.x/mark.html).

Reply:
Done.